compiler/aarch64/codegen/OMRInstOpCodeEnum.hpp

```diff
@@ -390,6 +390,8 @@
 		umsubl,                                                 	/* 0x9BA08000	UMSUBL    	 */
 		smulh,                                                  	/* 0x9B400000	SMULH     	 */
 		umulh,                                                  	/* 0x9BC00000	UMULH     	 */
+		fmaddd,														/* 0X1F000000   FMADD        */
+		fmadds,														/* 0X1F400000   FMADD        */
```

Comment:
The alignment of the comments looks wrong here.

Reply:
The file is using space but only the character right before the comment is using tab. I have fixed my change to align with the space/tab convention in the file.